lighthouse-core/test/report/html/renderer/psi-test.js

```diff
@@ -82,7 +82,6 @@ describe('DOM', () => {
         const result = prepareLabData(sampleResultsStr, document);
         assert.ok(result.scoreGaugeEl instanceof document.defaultView.Element);
         assert.equal(result.scoreGaugeEl.querySelector('.lh-gauge__wrapper').href, '');
-        assert.ok(result.scoreGaugeEl.outerHTML.includes('<style>'), 'score gauge comes with CSS');
```

Comment:
I assume this is ok? The PSI renderer already includes `report-styles.css`?

Reply:
pretty sure it does. @exterkamp  to confirm

Reply:
Yup, PSI imports report-styles.  Should be g2g!